ReText/__init__.py

```diff
@@ -59,6 +59,9 @@ def getBundledIcon(iconName):
     'font': '',
     'handleWebLinks': False,
     'hideToolBar': False,
+    'showToolBarFile': True,
+    'showToolBarEdit': True,
+    'showToolBarFormat': True,
```

Comment:
Maybe these names would sound more naturally in English?

```suggestion
    'showFileToolBar': True,
    'showEditToolBar': True,
    'showFormatToolBar': True,
```